### ISSUE TITLE
Did you intentionally decide to screw everyone?

### ISSUE THREAD

**Issue by elvisimprsntr** [dennisreimann/ioctocat#433]
I previously purchased the pro version.  Now with the most recent update you are forcing me to purchase another in app upgrade.   I tried to restore a previous upgrade and it states I have not purchased an upgrade with my Apple ID which is false.   

Running iOS 7.1b4

**Comment by dennisreimann** [comment]
Hello Elvis, no this isn't intentionally and should not happen. It might be an error or something with the iOS 7.1 beta, but you won't have to pay again for any of the upcoming 2.x versions.

**Comment by elvisimprsntr** [comment]
I am beta testing an app for another developer which also has in app purchases. He set a temp account in order to restore previous purchases so I can test those related features.   The restore previous purchases worked without a problem.  So it seems like it is not a problem with iOS 7.1b4. 

E

**Comment by elvisimprsntr** [comment]
I had previously intentionally logged out of my account to test another app.  When I was prompted by your app to enter my iTunes account information it correctly asked for both my ID and password, but it failed to restore my previous purchases.   

After reporting this issues last night, I used the iTunes settings to manually log back into my iTunes account. This morning when I launched the app I was once again prompted to enter only my iTunes password.  Now it seems like it automatically restored my previous purchases.  Still seems odd that your app automatically prompts for my iTunes account upon launch unless that is intentional upon first launch after an update.    Also seems odd that it could not restore previous purchases when not currently logged into my iTunes account unless someone did something last night to address the issue.
